src/main/resources/db/changelog/changes/14-add-precision-to-total-in-orders-table.yaml

```diff
@@ -0,0 +1,9 @@
+databaseChangeLog:
+  - changeSet:
+      id: add-precision-to-total-in-orders-table
+      author: oleksandr
+      changes:
+        - modifyDataType:

```

Comment:
auto rollback not supporter for such operation
remove this changeset and change data type in exists changelog where you create orders

Reply:
gtk, i thought that any changes to the existing scripts will lead to failure

Reply:
Correct, if you change existing changelogs and run app liquibase will fail, that's why you need to drop db. 
If you would like to modify data type you need to implement correct rollback